app/javascript/components/App.js

```diff
@@ -198,24 +268,43 @@ export default class App extends React.Component {
             path="/lended/:id"
             render={(props) => {
               let id = props.match.params.id
-              let book = this.state.books.find(book => book.id === parseInt(id))
-              return (
-                <LendedShow book={book} />
-              )
+              let book = this.findItem(this.state.books, parseInt(id))
+              if (book) {
+                return (
+                  <LendedShow
+                    book={book}
+                  />
+                )
+              } else {
+                return (
+                  <div>
+                    Loading Book You Lended...
+                  </div>
+                )
+              }
             }}
           />
 
           <Route path="/lend/:id"
             render={(props) => {
               let id = props.match.params.id
               let book = this.state.books.find(book => book.id === parseInt(id))
-              return (
-                <LendEdit
-                  updateBook={this.updateBook}
-                  current_user={current_user}
-                  book={book}
-                />
-              )
+              console.log(book)
```

Comment:
🔥  - Do you need this console log in here?

Reply:
Nope, thanks!